npm install fails due to missing lcms2 linking

Compiling on Ubuntu 14.04:

```
$ npm install

> janus-image-worker@0.1.1 postinstall /home/gaul/work/janus-image-worker
> node node/postinstall.js

✓ jpeg turbo downloaded successfully
✓ mozjpeg downloaded successfully
✓ jpeg-turbo built successfully.
✓ mozjpeg built successfully.
✗ Unable to build image recompressor.
src/pngquant/rwpng.o: In function `rwpng_read_image24_libpng':
rwpng.c:(.text+0xc3f): undefined reference to `cmsCreate_sRGBProfile'
rwpng.c:(.text+0xc5f): undefined reference to `cmsCreateTransform'
rwpng.c:(.text+0xca4): undefined reference to `cmsDoTransform'
rwpng.c:(.text+0xcb7): undefined reference to `cmsDeleteTransform'
rwpng.c:(.text+0xcbf): undefined reference to `cmsCloseProfile'
rwpng.c:(.text+0xcc7): undefined reference to `cmsCloseProfile'
rwpng.c:(.text+0xf3a): undefined reference to `cmsBuildGamma'
rwpng.c:(.text+0xf6f): undefined reference to `cmsCreateRGBProfile'
rwpng.c:(.text+0xf7f): undefined reference to `cmsFreeToneCurve'
rwpng.c:(.text+0xfb0): undefined reference to `cmsOpenProfileFromMem'
rwpng.c:(.text+0xfbb): undefined reference to `cmsGetColorSpace'
rwpng.c:(.text+0x1031): undefined reference to `cmsCloseProfile'
collect2: error: ld returned 1 exit status
make: *** [all] Error 1

npm ERR! weird error 1
npm WARN This failure might be due to the use of legacy binary "node"
npm WARN For further explanations, please read
/usr/share/doc/nodejs/README.Debian
 
npm ERR! not ok code 0
```

--- a/configure
+++ b/configure
@@ -28,7 +28,7 @@ for i in "$@"; do
 done
 
 CFLAGS="$CFLAGS -O3 -Wall -std=c99"
-LDFLAGS="$LDFLAGS src/pngquant/lib/libimagequant.a -lpng -ljpeg -lz -lm"
+LDFLAGS="$LDFLAGS src/pngquant/lib/libimagequant.a -lpng -ljpeg -lz -lm -llcms2"
 
 if [[ $OSTYPE =~ "darwin" ]]; then
   :

